persist/fs/seek.go

```diff
@@ -78,6 +78,17 @@ func NewSeeker(filePathPrefix string, bufferSize int, bytesPool pool.BytesPool)
 	}
 }
 
+// GetFileIds retrieves all the identifiers present in the configured file
```

Comment:
Yep, sounds good. 

I added em in because golint was unhappy, do we have a config to control the scope of what it runs on?